components/org.wso2.carbon.business.rules.web/src/components/auth/Login.jsx

```diff
@@ -94,7 +103,7 @@ export default class Login extends React.Component {
             .then(() => this.setState({authenticated: true}))
             .catch((error) => {
                 const errorMessage = error.response && error.response.status === 401 ?
-                    'The username/password is invalid' : 'Unknown error occurred!';
+                    'The username/password is invalid' : 'Unknown error occurred!';;
```

Comment:
Is it possible to check for other HTTP codes and provide error messages?

Reply:
Have followed the same approach as https://github.com/wso2/carbon-dashboards/blob/master/components/dashboards-web-component/src/auth/Login.jsx#L188.